src/frontends/onnx/README.md

```diff
@@ -0,0 +1,54 @@
+# ONNX FE
+
+The main responsibility of ONNX Frontend is import of ONNX models and conversion of these into the `ov::Model` representation. 
+Other capabilities of the ONNX Frontend:
+* modification of tensors properties (like data type and shapes)
+* changing topology of the models (like cutting subgraphs, inserting additional inputs and outputs)
+* user-friendly searching the models (via tensors and operators names)
+
+The component is written in `C++`, `Python` bindings are also available.
+Each change to ONNX FE component requires `clang-format` code style and `NCC` naming style checks.
+
+
+## Key contacts
+
+In case of any questions, review and merge requests, please contact:
+* Tomasz Dołbniak (tomdol)
+* Mateusz Tabaka (mateusztabaka)
+* Mateusz Bencer (mbencer)
+
+from `openvino-onnx-frontend-maintainers` maintainers group.
+
+
+## Components
+
+ONNX Frontend implements an interface common for all frontends defined in [Frontends API](https://github.com/openvinotoolkit/openvino/tree/master/src/frontends/common/include/openvino/frontend).
+The exported symbols are decorated with `ONNX_FRONTEND_API` macro.
+For backward compatibility reasons, the ONNX importer API (more lower-level abstraction approach) is still maintained. It can be found in (ONNX Importer)[https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/onnx/frontend/include/onnx_import/onnx.hpp]. The symbols of ONNX Importer API are decorated with `ONNX_IMPORTER_API` macro.
```

Comment:
It is low level details, I think it can be described on dedicated page inside the docs folder.
The idea is to don't overload user by information about the product, here we can have links to dedicated pages and he will read in case of needed.

Reply:
I see, I've removed some details (like exporting flags), but still keep some very basic info about op registering (to consider when tutorial - how to add a new op will be ready)

Reply:
Can we do this page now? :)
I added documentation for IR frontend: https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/ir/README.md

Maybe you can also take a look to this page. Here we don't provide any details on  the main page, for details dedicated pages are created

Reply:
done, dedicated page (TBD - 93001) created, details in the main page removed